### DISCUSSION TITLE
[Question] Phone Numbers inside DB

### DISCUSSION THREAD

**Discussion by patachi** [dcm4che/dcm4chee-arc-light#4439] [Q&A] [answered]
Are `(0040,1103) LO Person's Telephone Numbers` or `(0010,2154) SH Patient's Telephone Numbers` tags stored in the DB?
If yes, which table & column?
if not, is there any way to save them in the DB?

there are `study_custom1, study_custom2, study_custom3` columns inside the `study` table, Are there any configurations to tell dcm4chee to fill them with the value of specific dicom tag?

**Comment by vrindanayak** [comment, marked as answer]
These are patient level attributes and are supported in `Patient` - [Attribute Filter](https://dcm4chee-arc-cs.readthedocs.io/en/latest/networking/config/attributeFilter.html) by default. Introduced in https://github.com/dcm4che/dcm4chee-arc-light/issues/4220

**Comment by patachi** [comment]
@vrindanayak  thank you.
 
Here is my config using docker [[dcm4che/slapd-dcm4chee](https://hub.docker.com/r/dcm4che/slapd-dcm4chee)] for those who may encounter the same issue.


1. create a new ldif file e.g. `patient_custom_attributes.ldif`
```
version: 1

dn: dcmEntity=Patient,dicomDeviceName=dcm4chee-arc,cn=Devices,cn=DICOM Configuration,dc=dcm4che,dc=org
changetype: modify
add: dcmCustomAttribute1
dcmCustomAttribute1: DicomAttribute[@tag="00102154"]/Value[@number="1"]
-
add: dcmCustomAttribute2
dcmCustomAttribute2: DicomAttribute[@tag="00401103"]/Value[@number="1"]
-


```
2. mount/copy the `patient_custom_attributes.ldif` file into the docker container at `/etc/openldap/data/patient_custom_attributes.ldif` 

3. set or append to [IMPORT_LDIF](https://github.com/dcm4che-dockerfiles/slapd-dcm4chee/blob/master/README.md#import_ldif) environment variable your filename `patient_custom_attributes.ldif`
